src/main/python/measure_judged.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+'''
+Anserini: A Lucene toolkit for replicable information retrieval research
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import argparse
+import collections
+
+from typing import Dict
+from typing import List
+from typing import Set
+
+
+def load_qrels(path: str) -> Dict[str, Set[str]]:
+    """Loads qrels into a dict of key: query_id, value: set of relevant doc
+    ids."""
+    qrels = collections.defaultdict(set)
+    with open(path) as f:
+        for i, line in enumerate(f):
+            line = ' '.join(line.split())
+            query_id, _, doc_id, relevance = line.rstrip().split(' ')
+            qrels[query_id].add(doc_id)
+            if i % 1000000 == 0:
+                print('Loading qrels {}'.format(i))
```

Comment:
Tested this script on my local machine. My qrels file is short. Hence this script printed:
```
Loading qrels 0
judged@5: 0.2
judged@10: 0.15333333333333335
judged@20: 0.15666666666666662
judged@30: 0.14
Done
```
It threw me off a little bit when I see `Loading qrels 0` alone.